strings/charactes_unique.py

```diff
@@ -18,23 +27,31 @@ def isUniqueSet(s):
         hashSet.add(i)
     return True
 
-def isUniqueHash(s):
+
+def isUniqueHash(s: str) -> str:
     """
-    This uses Hashes
+    This function uses ascii values as index to determine
+    the string contains unique characters
     s -> string
```

Comment:
After having Type hints, there is no need for this.

Reply:
Sorry but Python 2 died on 1/1/2020.  One of the ways that Python 3 is different than Python 2 is that all str are Unicode.  Please include doctests that demonstrate that this code proper deals with sentences that include ãèíôü😀🎄